.github/workflows/fuzzing.yml

```
@@ -149,6 +153,10 @@
       run: |
         cmake --build build --config RelWithDebInfo
 
+    - name: Generate dictionary
+      run: |
```

actionlint

shellcheck reported issue in this script: SC1001:info:1:12: This \d will be a regular 'd' in this context

(shellcheck)